requirements.in

```diff
@@ -0,0 +1,2 @@
+aiohttp
+sqlalchemy
```

Comment:
OK. This was meant to be a fast PR just to get something to start with, so I will reduce the docker-compose.yml in order to get going.